### ISSUE TITLE
[Bug]: Judy

### ISSUE THREAD

**Issue by herauthon** [netdata/netdata#13677]
### Bug description

/usr/bin/ld: libnetdata/dictionary/dictionary.o: in function `dictionary_get_name_value_unsafe':
dictionary.c:(.text+0x1b5): undefined reference to `JudyHSGet'
/usr/bin/ld: libnetdata/dictionary/dictionary.o: in function `string_strdupz':
dictionary.c:(.text+0x697): undefined reference to `JudyHSGet'
/usr/bin/ld: dictionary.c:(.text+0x732): undefined reference to `JudyHSIns'
/usr/bin/ld: libnetdata/dictionary/dictionary.o: in function `dictionary_set_name_value_unsafe':
dictionary.c:(.text+0x87d): undefined reference to `JudyHSIns'
/usr/bin/ld: libnetdata/dictionary/dictionary.o: in function `string_freez':
dictionary.c:(.text+0xdb7): undefined reference to `JudyHSDel'
/usr/bin/ld: libnetdata/dictionary/dictionary.o: in function `dictionary_del_unsafe':
dictionary.c:(.text+0x3087): undefined reference to `JudyHSGet'
/usr/bin/ld: dictionary.c:(.text+0x30b9): undefined reference to `JudyHSDel'
/usr/bin/ld: libnetdata/dictionary/dictionary.o: in function `dictionary_destroy.part.0':
dictionary.c:(.text+0x36b9): undefined reference to `JudyHSFreeArray'
/usr/bin/ld: libnetdata/dictionary/dictionary.o: in function `thread_cache_entry_get_or_set':
dictionary.c:(.text+0x4d83): undefined reference to `JudyHSIns'
/usr/bin/ld: libnetdata/dictionary/dictionary.o: in function `thread_cache_destroy':
dictionary.c:(.text+0x4e34): undefined reference to `JudyHSFreeArray'
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:5653: netdatacli] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/beheer/netdata'
make[1]: *** [Makefile:7352: all-recursive] Error 1
make[1]: Leaving directory '/home/beheer/netdata'
make: *** [Makefile:3800: all] Error 2
 FAILED  ''

 ABORTED  Failed to build Netdata.

### Expected behavior

normal completion of build script

### Steps to reproduce

1. using install script
2. or
3. using, configure, make ...

### Installation method

from source

### System info

```shell
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="11"
/etc/os-release:VERSION="11 (bullseye)"
/etc/os-release:VERSION_CODENAME=bullseye
/etc/os-release:ID=debian
```

### Netdata build info

```shell
N/A
```

**Comment by MrZammler** [comment]
Hi @herauthon !

Can you please post the exact steps you're using and the exact commands? I'm guessing you are not using the kickstart script, right? Are you cloning from master?

**Comment by MrZammler** [comment]
https://github.com/netdata/netdata/pull/13703 may have helped for this as well.

@herauthon can you try a version with the above patch to check if it's ok now please?

**Comment by ilyam8** [comment]
Closing due to no response. @herauthon, hey 🙋‍♂️ The issue is (likely) fixed in #13703.
